src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx

```diff
@@ -116,27 +135,6 @@ describe.each([
   ['wallet does not support EIP-1559', false],
   ['wallet supports EIP-1559', true],
 ])('useFormSend (%s)', (_, walletSupportsEIP1559) => {
-  const useSelectorMock = jest.spyOn(reactRedux, 'useSelector')
-
-  beforeEach(() => {
-    useSelectorMock.mockReturnValue({
-      [formData[SendFormFields.AccountId]]: {
-        bip44Params: {
-          purpose: 44,
-          coinType: 60,
-          accountNumber: 0,
-        },
-      },
```

Comment:
Now mocked l.54 since we now use `selectPortfolioAccountMetadataByAccountId` in the decoupled `handleSend`